Collectors/DetailedCollector.py

```diff
@@ -267,6 +270,33 @@ def returnVO(self,fname):
        else:
             return "noVO"
 
+    def process_gstream_tpc(self, gstream, addr, sid):
+         for event in gstream.events:
+
+             self.metrics_q.put({'type': 'gstream_message_tpc', 'count': 1})
+             event["tpc_protocol"] = event.pop("TPC")
+             event["client"] = event.pop("Client")
+             event["xeq"] = event.pop("Xeq")
+
+             self.logger.info(event["xeq"])
+             self.logger.info(type(event["xeq"]))
+             event["xeq"]["begin_transfer"] = event["xeq"].pop("Beg")
+             event["xeq"]["end_trasnfer"] = event["xeq"].pop("End")
+             event["xeq"]["ip_version"] = event["xeq"].pop("IPv")
+             event["xeq"]["return_code"] = event["xeq"].pop("RC")
+             event["xeq"]["used_streams"] = event["xeq"].pop("Strm")
+             event["xeq"]["flow_direction"] = event["xeq"].pop("Type")
+             event["source"] = event.pop("Src")
+             event["destination"] = event.pop("Dst")
+             event["size"] = event.pop("Size")
+
+             if event["source"].startswith('/store') or event["source"].startswith('/user/dteam'):
+                  self.logger.info("Sending GStream TPC for "+self._wlcg_exchange_tpc)
```

Comment:
For log entries on every event, I'd set the level down to debug, and maybe log the contents instead?
```suggestion
                  self.logger.debug("Sending WLCG GStream TPC: %s", str(event))
```